### ISSUE TITLE
Does KSCrash support IPv6-only?

### ISSUE THREAD

**Issue by gcoder-io** [kstenerud/KSCrash#156]

**Comment by kstenerud** [comment]
It uses Apple's networking layer, so it supports whatever Apple supports.

**Comment by dmitruss** [comment]
Hi, Guys! Any update about IPv6 supporting?
Thanks.

**Comment by kstenerud** [comment]
If iOS supports ipv6, so does KSCrash. I'm using the Apple networking components.

**Comment by naftaly** [comment]
Yes, KSCrash supports IPv6-only networks. HTTP networking uses standard Apple APIs which handle IPv6.
